src/coupling.jl

```diff
@@ -393,3 +394,189 @@ function apply_coupling!(sys::System{D, G, T},
     end
     return recompute_forces
 end
+
+@doc raw"""
+    MonteCarloMembraneBarostat(pressure, temperature, boundary; n_steps=30,
```

Comment:
Update the arguments here, also briefly describe the arguments and the constraint on the two arguments in the docstring.

Reply:
Done.